inspirehep/modules/workflows/workflows/article.py

```diff
@@ -298,10 +298,19 @@
             merge_articles,
             IF(
                 has_conflicts,
-                halt_record(
-                    action='merge_approval',
-                    message='Submission halted for merging conflicts.'
-                ),
+                [
+                    create_ticket(
```

Comment:
what happens if the workflow is restarted? don't you need to do some gymnastics to avoid creating an extra ticket?

Reply:
If the workflow will be restarted then I assume we should create another ticket as conflict will appear again, and probably old one will be closed already. Chance for restarting when it's halted waiting for curation is quite small, and workflows do not allow easily to generate a ticket if previous one is closed and do nothing if previous one is still opened :/